gpustat.py

```diff
@@ -114,6 +114,24 @@ def utilization(self):
         return int(v) if v is not None else None
 
     @property
+    def power_usage(self):
+        """
+        Returns the GPU power usage in Watts,
+        or None if the information is not available.
+        """
+        v = self.entry['power.use']
```

Comment:
Actually I think it should be "power.draw". A convention here is that I am using the same column name as shown in `nvidia-smi --help-query-gpu`.